Casks/l/lidanglesensor.rb

```diff
@@ -0,0 +1,21 @@
+cask "lidanglesensor" do
+  version "1.0.3"
+  sha256 "3ee216765f12673527cea98d5509bcd5799333653030ac7f953e3c7fa3dd2cd2"
+
+  url "https://github.com/samhenrigold/LidAngleSensor/releases/download/#{version}/LidAngleSensor.app.zip"
+  name "LidAngleSensor"
+  desc "Utility to display MacBook lid angle and play creaking sound"
+  homepage "https://github.com/samhenrigold/LidAngleSensor"
+
+  # Documentation: https://docs.brew.sh/Brew-Livecheck
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
+
+  depends_on macos: ">= :big_sur"
```

Comment:
```suggestion
  depends_on macos: ">= :big_sur"
```

The default strategy checking the Git tags should be sufficient, since each tag corresponds to a release and upstream does not use pre-releases.